packages/core/contracts/common/test/TokenSender.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.6.0;
+pragma experimental ABIEncoderV2;
+
+import "../../common/implementation/ExpandedERC20.sol";
+
+contract TokenSender {
+    function transferERC20(
+        address tokenAddress,
+        address recipientAddress,
+        uint256 amount
+    ) public returns (bool) {
+        IERC20 token = IERC20(tokenAddress);
+        token.transfer(recipientAddress, amount);
+        return true;
```

Comment:
Just curious, why return true?

Reply:
returned true to validate that the returned params from the libraries can be seen from the DSProxy that use the libraries :)